test/js/player.js

```diff
@@ -239,6 +239,18 @@ suite('player', function() {
     assert.equal(p.startTime, null);
     assert.equal(p.currentTime, null);
   });
+  test('setting the playbackRate on the player of an AnimationGroup updates also all child players', function() {
+    var group = new AnimationGroup([
```

Comment:
This is failing on travis (https://travis-ci.org/web-animations/web-animations-next/builds/46975684) as `player.js` is part of `web-animations.js` and cannot include features from web-animations-next such as groups.

You'll need to rewrite this test without using groups, and add an additional test for `web-animations-next-player.js` in `test/js/group-player.js` (that file needs to be renamed @rjwright)